doc/media/openage/modpack_definition_file.md

```diff
@@ -0,0 +1,109 @@
+# Modpack Definition File
+
+The modpack definition file is the header file for [modpacks](modpacks.md).
+
+TOML is used as the configuration language.
+
+This document contains information about the parameters and sections that
+can be interpreted by the engine.
+
+## File Header
+
+The file starts with the comment:
+
+```
+# openage modpack definition file
+```
+
+Additionally, the following parameters have to be specified
+
+Parameter            | Data Type | Optional | Description
+---------------------|-----------|----------|------------
+`file_version`       | String    | No       | Version of the modpack definition file format.
+
+
+## [info] Section
+
+`[info]` contains general information about the modpack.
+
+Parameter          | Data Type     | Optional | Description
+-------------------|---------------|----------|------------
+`packagename`      | String        | No       | Name of the modpack.
+`version`          | String        | No       | Version number.
+`repo`             | String        | Yes      | Name of the repo where the package comes from.
+`alias  `          | String        | Yes      | Alias of the modpack.
+`title`            | String        | Yes      | Title used in UI.
+`description`      | String        | Yes      | Path to a file with a short description (max 500 chars).
+`long_description` | String        | Yes      | Path to a file with a detailed description.
+`url`              | String        | Yes      | Link to the modpack's website.
+`license`          | Array[String] | Yes      | License(s) of the modpack.
+
+
+## [assets] Section
+
+`[assets]` contains paths to assets in the modpack.
+
+Parameter      | Data Type      | Optional | Description
+---------------|----------------|----------|------------
+`include`      | Array[String]  | No       | List of paths to assets that should be mounted on load time. Paths are allowed to contain wildcards.
+`exclude`      | Array[String]  | Yes      | List of paths to assets that should be excluded from mounting. Paths are allowed to contain wildcards.
```

Comment:
how will they be mounted? i.e. what mount point names with what content?

Reply:
Well, this still needs to be decided. We can leave this as-is for now.